Common/Vulkan/VulkanMemory.cpp

```diff
@@ -278,7 +278,7 @@ bool VulkanDeviceAllocator::AllocateFromSlab(Slab &slab, size_t &start, size_t b
 
 	// Remember the size so we can free.
 	slab.allocSizes[start] = blocks;
-	slab.tags[start] = { tag, time_now(), 0.0f };
+	slab.tags[start] = { tag, (float)time_now_d(), 0.0f };
```

Comment:
Hm, should really make both of these doubles.  There aren't that many of these so it should be fine.

-[Unknown]

Reply:
I was worried about savestates and thought it couldn't matter much here (but got too lazy to actually check whether these are saved heh).

Reply:
Come to think of it, this being the Vulkan slab allocator, these definitely aren't saved. I'll change t.

Reply:
I've definitely erred on the side of savings things unnecessarily, but even I wouldn't save the slabs, heh.

Although I have been thinking of saving the framebufs for reverse debugging...

-[Unknown]